File: Embassat/View/Extras/CADEMExtrasViewController.m
//
//  CADEMExtrasViewController.m
//  Embassa't
//
//  Created by Joan Romano on 11/06/14.
//  Copyright (c) 2014 Crows And Dogs. All rights reserved.
//

#import "CADEMExtrasViewController.h"

@interface CADEMExtrasViewController ()

@property (nonatomic, strong) IBOutletCollection(UILabel) NSArray *titleLabels;
@property (nonatomic, strong) IBOutletCollection(UILabel) NSArray *bodyLabels;

@end

@implementation CADEMExtrasViewController

- (instancetype)init
{
    if (self = [super init])
    {
        self.title = @"EXTRES";
    }
    
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor whiteColor];

    [self.titleLabels enumerateObjectsUsingBlock:^(UILabel *label, NSUInteger idx, BOOL *stop) {
        label.font = [UIFont em_boldFontOfSize:16.0];
    }];
    [self.bodyLabels enumerateObjectsUsingBlock:^(UILabel *label, NSUInteger idx, BOOL *stop) {
        label.font = [UIFont em_romanFontOfSize:16.0];
    }];
}

@end

File: Embassat/View/Extras/CADEMExtrasViewController.swift
//
//  CADEMExtrasViewController.swift
//  Embassa't
//
//  Created by Joan Romano on 16/05/15.
//  Copyright (c) 2015 Crows And Dogs. All rights reserved.
//

import UIKit

public class CADEMExtrasViewController: CADEMRootViewController {
    
    @IBOutlet var titleLabels: [UILabel]?
    @IBOutlet var subtitleLabels: [UILabel]?
    @IBOutlet var bodyLabels: [UILabel]?

    override public func viewDidLoad() {
        super.viewDidLoad()
        
        title = "Extres"
        
        view.backgroundColor = UIColor.whiteColor()
        
        if let labels = titleLabels {
            for label in labels {
                label.font = UIFont.detailFont(ofSize:15.0)
            }
        }
        
        if let labels = subtitleLabels {
            for label in labels {
                label.font = UIFont.detailFont(ofSize: 20.0)
            }
        }
        
        if let labels = bodyLabels {
            for label in labels {
                label.font = UIFont.detailFont(ofSize: 15.0)
            }
        }
    }

}
